src/components/ProjectSettingsTabs/GeneralTab.tsx

```diff
@@ -11,21 +11,26 @@ import { GeneralTabProps } from "./types";
 const tab = ProjectSettingsTabRoutes.General;
 
 export const GeneralTab: React.FC<GeneralTabProps> = ({
-  data,
+  projectData,
   projectId,
+  repoData,
   useRepoSettings,
 }) => {
   const { getTabFormState, updateForm } = useProjectSettingsContext();
   const currentFormState = getTabFormState(tab);
 
-  const initialFormState = useMemo(() => gqlToSchema(data), [data]);
+  const initialFormState = useMemo(() => gqlToSchema(projectData || repoData), [
+    projectData,
+    repoData,
+  ]);
   usePopulateForm(initialFormState, tab);
 
   const onChange = useCallback(({ formData }) => updateForm(tab, formData), [
     updateForm,
   ]);
 
-  const { validDefaultLoggers } = data;
+  const validDefaultLoggers =
```

Comment:
should this be the reverse? first try to use the repodata and only use the project one if repo is empty?

Reply:
No, we want to use the project's value if it has been set and, if not, use the repo value. Ultimately though it doesn't matter for this field–it's very static data that is the [same in both places](https://github.com/evergreen-ci/evergreen/blob/main/graphql/resolvers.go#L414-L420).

Reply:
ah true and true.